src/Symfony/Component/Validator/Resources/translations/validators.en.xlf

```diff
@@ -354,6 +354,10 @@
                 <source>This collection should contain only unique elements.</source>
                 <target>This collection should contain only unique elements.</target>
             </trans-unit>
+            <trans-unit id="92">
+                <source>This value is not a valid timezone.</source>
+                <target>This value is not a valid timezone.</target>
```

Comment:
Dieser Wert ist keine gültige Zeitzone.

For the German file

Reply:
Great thanks :)